packages/manager/apps/catalog-revamp/src/components/Filters/FilterItem.tsx

```diff
@@ -35,6 +35,7 @@ const FilterItem: React.FC<FilterItemProps> = ({
         onOdsCheckedChange={(event: CustomEvent) =>
           onCheckboxChange(type, label, event)
         }
+        data-tracking={`manager_product_catalog::filter::${type}::${label}`}
```

Comment:
you don't need to add manager_product_catalog

Reply:
Done, removed.